### DISCUSSION TITLE
Access asset value

### DISCUSSION THREAD

**Discussion by simplyniceweb** [statamic/cms#6623] [Q&A] [answered]
Hi, i'm new with Statamic and antlers.

I have tried to dump the data and i see my handler: homeimage_image
![image](https://user-images.githubusercontent.com/5621415/187951857-be40d8b0-46cc-427d-9e80-87b2188df807.png)

How can i access the value: lob_cv_2.png?

**Comment by duncanmcclean** [comment]
You can use the `->raw()` method to return that array.

**Reply by simplyniceweb** [reply]
Hi thanks! How do you access that using antlers?

**Reply by duncanmcclean** [reply]
`{{ field | raw }}`

**Reply by simplyniceweb** [reply]
I have tried to: {{ homeimage_image|raw|dump }} gives me this result:
![image](https://user-images.githubusercontent.com/5621415/187963694-0c7f2601-16e0-4e0c-89b7-37726c9ab6df.png)

i can't seem to loop into it:
`        {{ homeimagex = homeimage_image|raw }}
        {{ foreach:homeimagex }}
            {{ image }}
        {{ /foreach:homeimagex }}`
i'm not getting any value.

**Reply by duncanmcclean** [reply]
You can do it like this:

```antlers
{{ field | raw }}
  {{ value }}
{{ /field }}
```

`{{ value }}` will give you the filenames like in the array. You won't have access to anything augmented (like the URL), since you're not getting the augmented value.

**Comment by jasonvarga** [comment, marked as answer]
Since asset fieldtypes get augmented into asset objects, you will have a bunch of different variables available in the loops.

The paths - which are what you're seeing in the dump - can be retrieved using the `path` variable.

```antlers
{{ homeimage_image }}
  {{ path }}
{{ /homeimage_image }}
```

See https://statamic.dev/fieldtypes/assets#templating

**Reply by simplyniceweb** [reply]
Thank you @jasonvarga !

Is there anyway to access the first path like: {{ homeimage_image[0]['path'] }}?

**Reply by duncanmcclean** [reply]
You should be able to get the first with `{{ homeimage_image:0:path }}`

**Reply by simplyniceweb** [reply]
Thanks @duncanmcclean !
